packages/utilities/src/utilities/global/global.js

```diff
@@ -0,0 +1,29 @@
+import { AnalyticsAPI } from '@carbon/ibmdotcom-services';
+import { DDOAPI } from '@carbon/ibmdotcom-services';
+
+/**
+ * Flag to determine if the global init has been fired
+ *
+ * @type {boolean}
+ * @private
+ */
+let _initialized = false;
+
+/**
+ * Initializes various global functions
+ */
+export function globalInit() {
+  if (_initialized) {
+    return;
+  } else {
+    _initialized = true;
+  }
+
+  // Sets the version of the library in the DDO
+  DDOAPI.setVersion();
+
+  // analytics tracking
+  AnalyticsAPI.initScrollTracker();
+  AnalyticsAPI.initDynamicTabs();
+  AnalyticsAPI.initModals();
```

Comment:
Does it make sense to add a method within `AnalyticsAPI` that does this? Something like `AnalyticsAPI.initAll()`?

Reply:
Updated in https://github.com/carbon-design-system/ibm-dotcom-library/pull/571/commits/421039ac7f4faef4256f586450b37ea2427a1f36